### DISCUSSION TITLE
Plugins for Kimai 2.0

### DISCUSSION THREAD

**Discussion by Rustleman** [kimai/Paid-plugins#87] [Questions & Answers]
I'm interested in these three plugins:
https://www.kimai.org/store/mr-software-controlling-bundle.html
https://www.kimai.org/store/mr-software-sickday-bundle.html
https://www.kimai.org/store/mr-software-vacation-bundle.html

Sadly they are not compatible with Kimai 2.0 and cannot be bought anymore.
Will there be anything comparable for the new Kimai version?

**Comment by pbksol** [comment]
Call yourself lucky that these are no longer sold. Don't ask for details ;-)

MR sent an e-mail to existing customers and told them that there will be somebody developing these plugins in the future, but no ETA or such. That's what I know.

**Reply by Rustleman** [reply]
OK ;-)
How difficult would it be to write new plugins from scratch with this functionality?

**Reply by kevinpapst** [reply]
See https://github.com/kimai/kimai/discussions/4145
